projects/convergence-rfq-solana/index.js

```diff
@@ -0,0 +1,29 @@
+
+const convergenceRfqSolEndpoint = 'https://mainnet.indexer.convergence.so/rfqs/trading-matrix'
+
+const dataFetcher = async () =>{
+    const response = await fetch(convergenceRfqSolEndpoint,{
+      method: 'GET',
+      headers: {
+        'Content-Type': 'application/json',
+        'apikey': 'indexer-key.855ed0656f1c31bb1dda57aa7bdc4353'
+      },
+    })
+    const jsonData = await response.json()
+    console.log('json',jsonData)
+    return jsonData
+}
+
+const tvl = async () =>{  
+    const data = await dataFetcher();
+    const tvlData = data.tradingVolume.spot.overall + data.tradingVolume.options.overall + data.tradingVolume.futures.overall
```

Comment:
just a heads up, we cant treat trading volume as tvl, tvl = funds locked in smart contract.

to track volume/fees: https://github.com/DefiLlama/dimension-adapters